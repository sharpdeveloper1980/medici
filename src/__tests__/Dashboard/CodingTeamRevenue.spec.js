import React, { Component } from "react";
import { create } from "react-test-renderer";
import Chart from 'react-apexcharts';
const CodingTeamRevenue = (props) => {
    const apexLineChartWithLables = {
        chart: {
            height: 380,
            type: 'line',
            zoom: {
                enabled: false,
            },
            toolbar: {
                show: false,
            },
        },
        colors:['#ffaf47', '#ffaf47', '#ffaf47'],
        dataLabels: {
            enabled: false,
            
        },
        stroke: {
            width: [3, 3],
            curve: 'smooth',
        },
        title: {
            text: '',
            align: 'left',
            style: {
                fontSize: '14px',
            },
        },
        grid: {
            row: {
                colors: ['transparent', 'transparent'], // takes an array which will be repeated on columns
                opacity: 0.2,
            },
            borderColor: '#f1f3fa',
        },
        
        xaxis: {
            categories: ['July 01 ,2020', ' ', ' ', ' ', ' ', ' ', ' ',' ',' ','Aug 01 ,2020'],
            title: {
                text: 'Month',
            },
        },
        yaxis: {
            title: {
                text: '',
            },
            labels: {
                formatter: function (value) {
                  return value + "$";
                }
              },
            min: 0,
            max: 1000,
        },
        legend: {
            position: 'top',
            horizontalAlign: 'right',
            floating: true,
            offsetY: -25,
            offsetX: -5,
        },
        responsive: [
            {
                breakpoint: 600,
                options: {
                    chart: {
                        toolbar: {
                            show: false,
                        },
                    },
                    legend: {
                        show: false,
                    },
                },
            },
        ],
    };
    var apexLineChartWithLablesData = [
        {
            data: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
        }
    ];
    return (
        <div>
                <div className="row">
                    <div className="col-sm-9">
                        <h6 style={{ fontSize:'12px' }} className="header-title mb-2 mt-1">Coding Team Cumulative</h6>
                    </div>
                    <div className="col-sm-3">
                        <select className="form-control form-control-sm" style={{ fontSize:'10px',height:'23px' }}>
                            <option>Last 30 Days</option>
                            <option>Last 60 Days</option>
                        </select>
                    </div>
                </div>
                <Chart
                    options={apexLineChartWithLables}
                    series={apexLineChartWithLablesData}
                    type="line"
                    className="apex-charts"
                />
        </div>
    );
}

describe("CodingTeamRevenue component", () => {
  test("Matches the snapshot", () => {
    const codingTeamRevenue = create(<CodingTeamRevenue />);
    expect(codingTeamRevenue.toJSON()).toMatchSnapshot();
  });
});