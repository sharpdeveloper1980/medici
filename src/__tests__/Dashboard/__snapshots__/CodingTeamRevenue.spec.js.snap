// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CodingTeamRevenue component Matches the snapshot 1`] = `
<div>
  <div
    className="row"
  >
    <div
      className="col-sm-9"
    >
      <h6
        className="header-title mb-2 mt-1"
        style={
          Object {
            "fontSize": "12px",
          }
        }
      >
        Coding Team Cumulative
      </h6>
    </div>
    <div
      className="col-sm-3"
    >
      <select
        className="form-control form-control-sm"
        style={
          Object {
            "fontSize": "10px",
            "height": "23px",
          }
        }
      >
        <option>
          Last 30 Days
        </option>
        <option>
          Last 60 Days
        </option>
      </select>
    </div>
  </div>
  <div
    className="apex-charts"
    height="auto"
    options={
      Object {
        "chart": Object {
          "height": 380,
          "toolbar": Object {
            "show": false,
          },
          "type": "line",
          "zoom": Object {
            "enabled": false,
          },
        },
        "colors": Array [
          "#ffaf47",
          "#ffaf47",
          "#ffaf47",
        ],
        "dataLabels": Object {
          "enabled": false,
        },
        "grid": Object {
          "borderColor": "#f1f3fa",
          "row": Object {
            "colors": Array [
              "transparent",
              "transparent",
            ],
            "opacity": 0.2,
          },
        },
        "legend": Object {
          "floating": true,
          "horizontalAlign": "right",
          "offsetX": -5,
          "offsetY": -25,
          "position": "top",
        },
        "responsive": Array [
          Object {
            "breakpoint": 600,
            "options": Object {
              "chart": Object {
                "toolbar": Object {
                  "show": false,
                },
              },
              "legend": Object {
                "show": false,
              },
            },
          },
        ],
        "stroke": Object {
          "curve": "smooth",
          "width": Array [
            3,
            3,
          ],
        },
        "title": Object {
          "align": "left",
          "style": Object {
            "fontSize": "14px",
          },
          "text": "",
        },
        "xaxis": Object {
          "categories": Array [
            "July 01 ,2020",
            " ",
            " ",
            " ",
            " ",
            " ",
            " ",
            " ",
            " ",
            "Aug 01 ,2020",
          ],
          "convertedCatToNumeric": true,
          "title": Object {
            "text": "Month",
          },
        },
        "yaxis": Object {
          "labels": Object {
            "formatter": [Function],
          },
          "max": 1000,
          "min": 0,
          "title": Object {
            "text": "",
          },
        },
      }
    }
    series={
      Array [
        Object {
          "data": Array [
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
            0,
          ],
        },
      ]
    }
    type="line"
    width="100%"
  />
</div>
`;
